frontend/src/components/SettingsGeneral.vue

```diff
@@ -1,18 +1,22 @@
 <script setup lang="ts">
 import { ColorSchemes } from '@/definitions';
 import {
-  ref, reactive, inject, watch,
+  ref, reactive, inject, watch, computed,
 } from 'vue';
 import { useI18n } from 'vue-i18n';
 import { useUserStore } from '@/stores/user-store';
-import { dayjsKey, callKey } from '@/keys';
+import { dayjsKey, callKey, isoWeekdaysKey } from '@/keys';
 import { SubscriberResponse } from '@/models';
 
 // component constants
 const user = useUserStore();
 const { t, locale, availableLocales } = useI18n({ useScope: 'global' });
 const call = inject(callKey);
 const dj = inject(dayjsKey);
+const isoWeekdays = inject(isoWeekdaysKey);
+
+const startOfTheWeek = ref(localStorage?.getItem('startOfTheWeek') ?? isoWeekdays[0].long.toLowerCase());
+const availableStartOfTheWeekOptions = computed(() => isoWeekdays.filter((day) => day.long === 'Monday' || day.long === 'Sunday'));
```

Comment:
This won't work for other languages than English. E.g. when viewing the settings page in German, this array would be empty.